rootski_db_migrations/src/migrations/rootski_db/versions/4_add_word_translations.py

```diff
@@ -0,0 +1,56 @@
+# pylint: disable=invalid-name
+# noqa: D400
+"""
+Migration for adding a ``translation`` column and seed data into the ``words`` table.
+
+Revision ID: 4
+Revises: 3
+Create Date: 2022-03-20 21:16:47.804931
+
+"""
+import json
+
+import pandas as pd
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "4"
+down_revision = "3"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    """Add a ``translation`` column to the ``words table."""
+    op.add_column(
+        table_name="words",
+        column=sa.Column(
+            "translation",
+            sa.String(512),
+            nullable=True,
+            comment="a rough english translation for searching english words",
+        ),
+    )
+
+    # Empty the words table
+    # Allows us to empty the words table by bypassing the foreign key constraints
+    op.execute("SET session_replication_role = 'replica';")
+    op.execute("DELETE FROM WORDS;")
+
+    # Get the words data including translations
+    words_df = pd.read_csv("migrations/initial_data/data/words_with_translations.csv")
```

Comment:
Will this file path always work? Could you do the `THIS_DIR = Path(__file__).parent` trick and also make this file path a constant at the top of the file?

I'm planning to merge this in before that so others can already start using this.